gravitee-apim-definition/gravitee-apim-definition-model/src/main/java/io/gravitee/definition/model/v4/Api.java

```diff
@@ -72,6 +72,9 @@ public class Api implements Serializable {
     @NotBlank
     private ApiType type;
 
+    @JsonProperty(required = false)
+    private String description;
+
```

Comment:
Is it absolutely required to add the description in this model?

gravitee-apim-definition contains the model that is loaded by the gateway. And the gateway doesn't care about the description.